modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -39,40 +27,32 @@ export class {{classname}} extends runtime.BaseAPI {
      * {{&summary}}
      {{/summary}}
      */
-    async {{nickname}}Raw({{#allParams.0}}requestParameters: {{operationIdCamelCase}}Request{{/allParams.0}}): Promise<runtime.ApiResponse<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}void{{/returnType}}>> {
-        {{#allParams}}
-        {{#required}}
-        if (requestParameters.{{paramName}} === null || requestParameters.{{paramName}} === undefined) {
-            throw new runtime.RequiredError('{{paramName}}','Required parameter requestParameters.{{paramName}} was null or undefined when calling {{nickname}}.');
-        }
-
-        {{/required}}
-        {{/allParams}}
+    async {{nickname}}Raw({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}{{#hasMore}}, {{/hasMore}}{{/allParams}}): Promise<runtime.ApiResponse<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}void{{/returnType}}>> {
         const queryParameters: runtime.HTTPQuery = {};
 
         {{#queryParams}}
         {{#isListContainer}}
-        if (requestParameters.{{paramName}}) {
+        if ({{paramName}}) {
```

Comment:
this could lead to many naming conflicts with symbols in the scope of the file

Reply:
I'm in the process of proposing a new version of this PR based on @TiFu comment on the issue. Should push this today! :)
Thanks for the review